frontend/src/components/header/userMenu/confirmLogOut/styles.css

```diff
@@ -0,0 +1,3 @@
+.confirm-button {
+    background-color: #657FB4;
```

Comment:
as I said before, manual approach is shit. Button colors must be unified https://www.notion.so/Sync-updated-design-6019f820711e4752ab2d237735df7db8